sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Add a Conclusion summarizing what was accomplished and suggested next steps.</p><p> </p><p>Are there any resources to delete in a Cleanup section? Hopefully the inference recommender jobs don't leave any open endpoints to delete?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
Added details in "Detailing out the result" section. No need to clean anything as recommender jobss don't leave any open endpoints.